### DISCUSSION TITLE
Generate Collection from existing JSON Schema

### DISCUSSION THREAD

**Discussion by SelfhostedPro** [payloadcms/payload#9367] [Feature Requests & Ideas]
Hey, it would be incredibly helpful to be able to generate a collection from an existing json schema (probably not within the scope of this project but figured I would make the request). That way I don't have to go through and define everything a second time.

Just going off of the current default "Media" collection and trying your best with fields would be great. Definitely understand if that's out of scope though.

**Comment by tyteen4a03** [comment]
Do you have a *lot* of collections to create...? Creating Collections is a one-time cost.

Maybe you meant you wanted to generate JSON schema from the Collection config?

**Reply by SelfhostedPro** [reply]
It’s just a really long schema I don’t want to have to deal with rewriting because I’m lazy ☹️ I’ll just get AI to do it though 😞
